src/protobuf-net.BuildTools/Internal/CodeGenSemanticModelParser.Namespace.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace ProtoBuf.Internal;
+
+internal static partial class CodeGenSemanticModelParser
+{
+    private static string _namespaceName;
```

Comment:
almost certainly shouldn't be static

Reply:
now we have a parse context on a namespace lvl. having that value there